design-documents/graph-ql/coverage/customer-email-password-update.md

```diff
@@ -15,7 +15,7 @@ Deprecate `updateCustomer` mutation in favor of `updateCustomerV2`. `CustomerUpd
 
  ```graphql
  mutation {
-    updateCustomerV2(input: CustomerUpdateInput!): Customer
+    updateCustomerV2(input: CustomerUpdateInput!): CustomerOutput
```

Comment:
I agree that it would not be consistent with existing operations. But I would only change it if we think that we may want to add more data in the future.